uwrt_mars_rover_utils/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>uwrt_mars_rover_utils</name>
+  <version>0.0.0</version>
+  <description>UWRT Mars Rover utilities package for avoiding duplicate code.</description>
+  <maintainer email="keyonjerome@gmail.com">keyonjerome</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
add the test_depends based off the readme template or off of what i configured for drivetrain